lib/services/service_brokers/service_manager.rb

```diff
@@ -58,16 +58,34 @@ def update_or_create_plans(catalog)
           plan.public = false
         end
 
+        create_instance = nil
+        begin
+          create_instance = catalog_plan.schemas.service_instance.create.schema.try(:to_json)
+        rescue
+        end
```

Comment:
There's a potential for exceptions to be lost here -- in the old code they would have been propagated to the top. Here the exceptions should be logged.

Also, in this block there's no need for the `.try` component.